### PR TITLE
-moz-submit-invalid behind a flag

### PR DESCRIPTION
**Pull request by rachelandrew**
Working on https://github.com/mdn/content/issues/3446

This feature is now behind a flag. I wasn't sure if I should do a separate block with version_removed for the unflagged version. If so let me know and I'll update it.

### CODE REVIEW

**Comment by ddbeck** [comment]
> I wasn't sure if I should do a separate block with version_removed for the unflagged version. If so let me know and I'll update it.

Yes, two support statements would be ideal. The first from version 88 with the flag data (i.e., an open-ended `"version_added": "88"`) and the second without the flag (i.e., from version 4 and closed-ended with `"version_removed": "88"`). Thank you!

**Comment by rachelandrew** [comment]
@ddbeck I've made that change

File: css/selectors/-moz-submit-invalid.json [hunk 1]
```diff
@@ -15,12 +15,40 @@
...
-            "firefox": {
-              "version_added": "4"
-            },
-            "firefox_android": {
-              "version_added": "4"
-            },
+            "firefox": [
+              {
+                "version_added": "88",
+                "notes": "From Firefox 88 the feature has been placed behind a flag. See <a href='https://bugzil.la/1694129'>bug 1694129</a>.",
```
Comment (ddbeck) [inline css/selectors/-moz-submit-invalid.json:21]: Doing something little remarkable here, so leaving a note for posterity: ordinarily, I'd say to leave this note out (or to only have the `See bug` portion). But this is an odd case of something going behind a pref after being unpreffed, so I'll leave it as is.

File: css/selectors/-moz-submit-invalid.json [hunk 2]
```diff
@@ -15,12 +15,40 @@
...
+                ]
+              },
+              {
+                "version_added": "4",
+                "version_removed": "88"
+              }
+            ],
+            "firefox_android": [
+              {
+                "version_added": "88",
```
Comment (ddbeck) [inline css/selectors/-moz-submit-invalid.json:37]: Oh, I'm sorry, @rachelandrew but I just remembered something significant: I learned last week that Firefox for Android doesn't provide access to preferences in regular releases (see https://github.com/mdn/browser-compat-data/issues/9679). Can we remove this support statement?

If you still think we ought to provide the bug link, we could add a note to the other support statement saying something like, `From version 88, the feature has been withdrawn. See <a href='https://bugzil.la/1694129'>bug 1694129</a>.`

**Review by ddbeck — CHANGES_REQUESTED** [review]

File: css/selectors/-moz-submit-invalid.json (hunk 2 above)
Comment (chrisdavidmills) [reply css/selectors/-moz-submit-invalid.json:37]: @rachelandrew hi! Can you finalize this one?

**Review by ddbeck — APPROVED** [review]
Thank you! 🎉

**Pull request merged**